docs/Getting Started/Adapting to the Head Unit Language/index.md

```diff
@@ -67,3 +68,74 @@ func managerShouldUpdateLifecycle(toLanguage language: SDLLanguage) -> SDLLifecy
     return configurationUpdate
 }
 ```
+!@
+
+@![android. javaSE, javaEE]
+
+`//TODO The closest thing I could find to this section in Android is "Handling a Language Change", should Android align this closer or should this section be placed here or should this sections be iOS and Android specific`
```

Comment:
Looks like Android has not implemented a change registration manager so the two documentations will be different. You can remove this TODO.

Some of the platform tags seem to be mismatched and are showing up on the webpage